SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/AbstractWorkerCoordinator.cs

```diff
@@ -51,20 +60,28 @@ protected Process CreateSimulatedPlayerProcess(string fileName, string args)
 
         /// <summary>
         /// Blocks until all active simulated player processes have exited.
+        /// Restarts failed simulated player processes.
         /// Will only wait for processes started through CreateSimulatedPlayerProcess().
         /// </summary>
         protected void WaitForPlayersToExit()
         {
-            while (ActiveProcesses.Count > 0)
+            while (true)
```

Comment:
Is `while(true)` OK? A possible way to change this would be to do 
```
while(!ActiveProcesses.All(process => process.HasExited && process.ExitCode == 0))
```
but this would cause a double iteration since we need the same when finding incorrectly finished processes.

Reply:
I think its fine as while(true)